docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc

```diff
@@ -45,6 +45,18 @@ finer, more granular time buckets, which are ideal for this type of analysis. If
 your detectors use <<ml-count-functions,count>> or <<ml-rare-functions,rare>>
 functions, set the interval to the same value as the bucket span.
 
+If your <<aggs-dfeeds,{dfeed} uses aggregations with nested `terms`>> and model 
+plot is not enabled for the {anomaly-job}, then neither can the **Single Metric 
+Viewer** nor the **Anomaly Explorer** in {kib} plot and display an anomaly 
```

Comment:
IMO we don't need to point out they're in Kibana here, but am fine with being overruled
```suggestion
Viewer** nor the **Anomaly Explorer** can plot and display an anomaly 
```